common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_apmanager.yaml

```diff
@@ -184,6 +184,23 @@ cACTION_APMANAGER_STEERING_EVENT_AUTH_FAIL_NOTIFICATION:
   _type: class
   params: sSteeringEvAuthFail
 
+cACTION_APMANAGER_WIFI_CREDENTIALS_SET_REQUEST:
+  _type: class
+  ssid:
+    _type: char    
+    _length: [ "beerocks::message::WIFI_SSID_MAX_LENGTH" ]
+  pass:
+    _type: char    
+    _length: [ "beerocks::message::WIFI_PASS_MAX_LENGTH" ]
+  auth_type:
+    _type: char    
+    _length: [ "beerocks::message::WIFI_SECURITY_TYPE_MAX_LENGTH" ]
+  enc_type:
+    _type: char    
+    _length: [ "beerocks::message::WIFI_SECURITY_TYPE_MAX_LENGTH" ]
+  is_backhaul: uint8_t
+  teardown: uint8_t
```

Comment:
Teardown should be a separate action. If we receive a teardown, that should be the only M2 we receive; anything else is an error.

Reply:
I agree but the its another set of messages:/

Reply:
Changed to list of configdata.